Tests/NetCore.Specs/NetCore.Specs.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Chill" Version="4.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
In Tests/Net47.Specs/Net47.Specs.csproj you referenced 16.3.0.

Reply:
Well spotted 👍 
For some reason (probably a mistake), 16.3.0 requires netcoreapp2.1.
I've reported it [here](https://github.com/microsoft/vstest/issues/2188)

For the TestFrameworks I upgraded to netcoreapp2.1, for the tests of the core library I updated to the currently latest supported version of `Microsoft.NET.Test.Sdk`.